packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-output.vue

```diff
@@ -9,12 +9,20 @@
 			<Button label="Save for reuse" outlined severity="secondary" />
 		</div>
 	</header>
+
 	<Accordion multiple :active-index="[0, 1, 2, 3]">
 		<AccordionTab header="Summary"> Summary text </AccordionTab>
 		<AccordionTab>
 			<template #header> State variables<i class="pi pi-info-circle" /> </template>
+			<!--TODO: Will put these checkbox options in output settings later-->
+			<div class="flex align-items-center gap-2 ml-4 mb-3">
+				<Checkbox v-model="onlyShowLatestBox" binary @change="renderCharts" /><label>Only show latest box</label>
+			</div>
```

Comment:
I think the language here needs a bit of tweaking? I don't see anywhere where we explained what `latest box` means. 

Also this slotted in the "state variables" section, but the effect of showing all boxes also modifies the parameter's section.

Reply:
Look at `getBoxesEndingAtLatestTimestep()` in `funman-service.ts`. This just shows the lines that make it to the furthest timestep.

It's slotted in the state variables section for now, I was thinking I would add these checkboxes to an Output Settings slider in another pass

Reply:
Maybe my first point wasn't clear, I meant since we never introduced the term "box" anywhere (that I know of), having a checkbox "only show latest box" is at best mysterious and at worst confusing. 

It is also conceivably possible to have multiple boxes at furtherest timestamp. For example at t=150 there are 2 true boxes where beta > 0.4 and beta > 0.2, and there is a false box for beta < 0.2.

So I dunno if your parsing logic needs to change, but "show only the last box" reads weird and can use a bit of rephrasing. How about "show only results with furtherest progress" or something akin to that?

Reply:
I changed the label. Parsing logic accounted for multiple so that's fine.